Server/db/MongoDB.js

```diff
@@ -195,7 +195,9 @@ const getAllMonitors = async (req, res) => {
 const getMonitorById = async (req, res) => {
   try {
     const monitor = await Monitor.findById(req.params.monitorId);
-    return monitor;
+    const checks = await Check.find({ monitorId: monitor._id });
+    const monitorWithChecks = { ...monitor.toObject(), checks };
+    return monitorWithChecks;
```

Comment:
Should we send a response to the client here?
```res.status().json()```

Reply:
Responses are sent back to thee server in the Controller, but yep that's the correct format 💯!

The DB returns its response to the controller which in turn returns to the client